package.json

```diff
@@ -21,6 +21,8 @@
     "check-coverage": "istanbul check-coverage --config .istanbul.yml",
     "upload-coverage-codacy": "cat ./coverage/lcov.info | ./node_modules/.bin/codacy-coverage",
     "upload-coverage-coveralls": "cat ./coverage/lcov.info | coveralls",
+    "artillery": "artillery run ./test/perf/all-results.json -o ./reports/all-results.json",
```

Comment:
First time I ran this I hadn't started localhost and also got an error at the end of the performance test run as the _reports_ directory didn't exist.

Resolved these issues by:

`npm install mkdirp --save-dev` and 

amending the artillary line to:

`"artillery": "mkdirp reports && npm start & artillery run ./test/perf/all-results.json -o ./reports/all-results.json",`

Note: the single _&_ runs commands concurrently